docs/contents/tutorials/02_migration_from_json_to_avro.rst

```diff
@@ -1,100 +0,0 @@
-Migration from JSON to AVRO
```

Comment:
maybe not, but its not valid anymore (with the addition of 1:N relation between Hermes and Kafka topics which is not on by default); lets add this once again in another pr